### PR TITLE
MCR-3021 Remove mapping of ORCID common:language-code

### PR DESCRIPTION
**Pull request by chemicus**
[Link to jira](https://mycore.atlassian.net/browse/MCR-3021).

### CODE REVIEW

**Comment by chemicus** [comment]
The reported error is due to a xsd-validation failure on the transformed orcid work. According to the scheme https://github.com/ORCID/orcid-model/blob/733021de324030e66bed0e4c821dc8cd30a3ef00/src/main/resources/record_3.0/work-3.0.xsd#L96  only one common:language-code is allowed in the scheme. This makes sense, since this field is meant to store the “language used for the metadata” https://github.com/ORCID/orcid-model/blob/733021de324030e66bed0e4c821dc8cd30a3ef00/src/main/resources/record_3.0/work-3.0.xsd#L98 (and not the language of the work/publication itself).
Therefore the correct mapping of common:language-code is mods:recordInfo/mods:languageOfCataloging rather than mods:language

File: mycore-orcid2/src/main/resources/xsl/orcid2/v3/work2mcr_generic.xsl [hunk 1]
```diff
@@ -208,11 +208,13 @@
...
   <xsl:template match="common:language-code">
-    <mods:language>
-      <mods:languageTerm authority="rfc5646" type="code">
-        <xsl:value-of select="document(concat('language:',text()))/language/@xmlCode" />
-      </mods:languageTerm>
-    </mods:language>
+    <mods:recordInfo>
+      <mods:languageOfCataloging>
+        <mods:languageTerm authority="rfc5646" type="code">
+          <xsl:value-of select="document(concat('language:',text()))/language/@xmlCode" />
```
Comment (toKrause) [inline mycore-orcid2/src/main/resources/xsl/orcid2/v3/work2mcr_generic.xsl:214]: Should check if a corresponding language classification value exists and handle the case when it doesn't.

**Review by toKrause — CHANGES_REQUESTED** [review]

File: mycore-orcid2/src/main/resources/xsl/orcid2/v3/work2mcr_generic.xsl (hunk 1 above)
Comment (toKrause) [inline mycore-orcid2/src/main/resources/xsl/orcid2/v3/work2mcr_generic.xsl]: Should check if a corresponding language classification value exists and handle the case when this is not the case.

**Review by kkrebs — CHANGES_REQUESTED** [review]
Currently there doesn't seem to be a suitable field for mods:language in ORCID, see https://support.orcid.org/hc/en-us/articles/360006971353-Metadata-in-the-Works-section

That's why we decided to leave it out completely.

**Review by kkrebs — APPROVED** [review]

**Review by toKrause — APPROVED** [review]

**Pull request merged**